ArduPlane/ArduPlane.cpp

```diff
@@ -150,6 +153,37 @@ constexpr int8_t Plane::_failsafe_priorities[7];
 constexpr int8_t Plane::_failsafe_priorities[6];
 #endif
 
+void Plane::update_state() {
+
+	float roll = ahrs.get_roll();
+	float pitch = ahrs.get_pitch();
+	float yaw  = ahrs.get_yaw();
+
+	Vector3f omega = ahrs.get_gyro();
+	
+	Vector3f velocity;
+	ahrs.get_velocity_NED(velocity);
+
+	Vector3f position;
+	ahrs.get_relative_position_NED_origin(position);
+
+	printf("Position: %.3f, %.3f, %.3f\n", position.x, position.y, position.z);
+	printf("Velocity: %.3f, %.3f, %.3f\n", velocity.x, velocity.y, velocity.z);
+	printf("Angles:   %.3f, %.3f, %.3f\n", roll*3.14/180, pitch*3.14/180, yaw*3.14/180);
```

Comment:
Are we sure we need these conversions? I can't seem to find get_roll() in ahrs so I can not confirm.